docs/developer-docs/multi-chain/ethereum/using-eth/eth-dev-workflow.mdx

```diff
@@ -19,7 +19,7 @@ To interact with the Ethereum network or another EVM-compatible network, a devel
 
 1. [Generate a public key](generating-addresses.mdx): To sign and submit transactions, your canister will need a public key. You will also need this public key to generate an Ethereum wallet address.
 
-2. [Sign a transaction](signing-transactions.mdx): Next, a transaction must be signed before it can be submitted to the Ethereum network and executed. Transactions are signed using a canister's public key.
+2. [Sign a transaction](signing-transactions.mdx): Next, a transaction must be signed before it can be submitted to the Ethereum network and executed. Transactions are signed using [threshold ECDSA](/docs/current/references/t-ecdsa-how-it-works), a novel protocol as part of ICP's chain-key signatures suite. Transactions are signed using a canister's private key.
```

Comment:
```suggestion
2. [Sign a transaction](signing-transactions.mdx): Next, a transaction must be signed before it can be submitted to the Ethereum network and executed. Transactions are signed using [threshold ECDSA](/docs/current/references/t-ecdsa-how-it-works), a novel protocol as part of ICP's chain-key signatures suite.
```